### ISSUE TITLE
low Accuracy

### ISSUE THREAD

**Issue by Zumbalamambo** [blackeagle01/Abnormal_Event_Detection#4]
I have trained avenue dataset for about 1000 epoch but still the accuracy is just 76% . Will it improve the accuracy if I increase the epoch further more? However the accuracy doesnt seem to be increasing. Can you also please update realtime video prediction code ?

**Comment by blackeagle01** [comment]
Try tweaking the batch size and the threshold parameters in the train and test modules for better accuracy.

**Comment by blackeagle01** [comment]
I'll update the real time prediction code as soon as my exam week ends.

**Comment by Zumbalamambo** [comment]
@blackeagle01 sure... waiting...

**Comment by blackeagle01** [comment]
@Zumbalamambo added the live feed feature. Kindly test it with your trained model.

**Comment by Zumbalamambo** [comment]
let me give it a try and will update you about how it goes...thank you!

**Comment by Zumbalamambo** [comment]
the video feed is visible only after processing. Is it possible to show video feed while classifying?

**Comment by blackeagle01** [comment]
@Zumbalamambo will check on it.

**Comment by blackeagle01** [comment]
@Zumbalamambo can you send me the trained model file so that I can perform optimisations! Share it on google drive.

**Comment by Zumbalamambo** [comment]
Please Use [this](https://drive.google.com/open?id=15byLKcV8ZztZqxB--Pxf_ZsyLUpcQbU7) link to download the model.

**Comment by Zumbalamambo** [comment]
@blackeagle01 can it show the feed while predicting?

**Comment by blackeagle01** [comment]
@Zumbalamambo working on it.

**Comment by PBRAOS** [comment]
> I have trained avenue dataset for about 1000 epoch but still the accuracy is just 76% . Will it improve the accuracy if I increase the epoch further more? However the accuracy doesnt seem to be increasing. Can you also please update realtime video prediction code ?

Did you manage to get a better accuracy ? I have the same issue on the code...

**Comment by Zumbalamambo** [comment]
@PBRAOS yes, Finally, I managed to get 98.3% accuracy in the Test dataset. I changed a few hyper params, layers etc.., and it worked.

**Comment by PBRAOS** [comment]
Could you please share the main changes you made if you can remember ?

**Comment by PBRAOS** [comment]
> @PBRAOS yes, Finally, I managed to get 98.3% accuracy in the Test dataset. I changed a few hyper params, layers etc.., and it worked.

Could you please share the main changes you made if you can remember ?

**Comment by PBRAOS** [comment]
> @PBRAOS yes, Finally, I managed to get 98.3% accuracy in the Test dataset. I changed a few hyper params, layers etc.., and it worked.

@Zumbalamambo Could you please share the main changes you made if you can remember ?

**Comment by AndyHon** [comment]
@Zumbalamambo Can you upload your changes to your GitHub? Thank you

**Comment by PBRAOS** [comment]
I sincerely challenge one can have that precision...

Pb

**Comment by Zumbalamambo** [comment]
@PBRAOS @AndyHon Unfortunately I can not share it since I did this for the company that I have made my contract. Sorry about that.

**Comment by alchemi5t** [comment]
@Zumbalamambo did you write a script to calculate the accuracy? cause there is'nt one in this repo.

**Comment by deepakkrish44** [comment]
I hope accuracy is not the issue here... How much AUC (Area under the curve) results are you getting for both ped1 and ped2.

1) The model provided here gives only 70-75 % AUC for both ped2 and avenue datasets.
2) I have trained a model from scratch using the same architecture in the paper and have also performed data augmentation as it is given in the paper ( strides). But still, AUC values are not reaching even 80%.
3) How does hyperparameter tuning achieve groundbreaking results like 90% when the loss of the model saturates after 30 to 40 epochs. (min loss obtained - 0.064..) but still testing performance is poor.
Any idea on it would be great. 
Sorry if I am wrong,
thank you

**Comment by deepakkrish44** [comment]
@PBRAOS shall I know in which dataset you have obtained 98 percent accuracy?
